dpgen/collect/collect.py

```diff
@@ -24,8 +24,12 @@ def collect_data(target_folder, param_file, output,
     init_data = []
     init_data_prefix = jdata.get('init_data_prefix', '')
     init_data_sys = jdata.get('init_data_sys', [])
-    for ii in init_data_sys:
-        init_data.append(dpdata.LabeledSystem(os.path.join(init_data_prefix, ii), fmt='deepmd/npy'))        
+    for ii in init_data_sys:
```

Comment:
CI says this line has an invalid character in identifier. Please make sure CI passes.
https://travis-ci.org/github/deepmodeling/dpgen/jobs/727892051